app/views/posts/show.html.erb

```diff
@@ -26,7 +27,13 @@
       <% end %>
     </ul>
   </div>
-
 <div class="btn">
-    <%= link_to "Pagination", user_posts_path(@users), class: 'button _center'%>
+<%= form_with model: @like, url: user_post_likes_path(post_id: @post.id) do |form| %>
+  <span><%= form.submit "Like", class: 'button _center' %></span>
```

Comment:
- [ ] I could not check if the like button works because of the errors on this page, so kindly ensure it is functional.